src/constants/plans.ts

```diff
@@ -5,6 +5,13 @@ export const PLANS = {
     enterprise: 'Enterprise',
   },
   cols: [
+    {
+      rows: '1',
+      isRowTitle: true,
```

Comment:
```suggestion
      rows: '3',
      isRowTitle: false,
```
A little bit of explanation on how these properties work: `rows `can be 1, 2 or 3 only. This is necessary to set the height of string on adaptives (the bigger the higher). For rows with a large amount of text (like this one with the long `enterprise `description), specify 3. If you set it to 1, you will get this content error:
![image](https://github.com/Avaiga/website/assets/88053873/7ad2cd52-830a-4ebb-8f05-59e259ace7e2)

isRowTitle value is set to true only for rows that have no content ( = ```
сommunity: '',
      enterprise: ''
```), in other words are subheadings. Such `rows `are hidden on sm resolution. So if you will set `isRowTitle: true,` for row with content, it will dissapear on mobiles:
![image](https://github.com/Avaiga/website/assets/88053873/e868d792-dd59-4bc9-abcf-06c241b4abcd)

Reply:
nope @voolga I did wante it to be a title and at the first row, so no need to change these